src/controllers/conversation.js

```diff
@@ -0,0 +1,32 @@
+import { sendDataResponse } from '../utils/responses.js'
+import Conversation from '../domain/conversation.js'
+
+export const createConversation = async (req, res) => {
+  const { name, createdBy, usersIds } = req.body
+
+  try {
+    if (!name || usersIds.length === 0 || !createdBy) {
+      throw new Error('Please provide appropriate content')
+    }
+    const conversationToCreate = await Conversation.fromJson(req.body)
+    const conversation = await conversationToCreate.save()
+    return sendDataResponse(res, 201, conversation)
+  } catch (err) {
+    return sendDataResponse(res, 400, { err: err.message })
+  }
+}
+
+export const findAllConversationsByUserId = async (req, res) => {
+  const { userId } = req.params
+
+  try {
+    const conversations = await Conversation.findAll(userId)
+    if (conversations.length === 0) {
```

Comment:
This is a non-blocking comment (so feel free to not make any changes)

checking the length of an array is 0 isn't an error case - returning an empty array is expected if there are no conversations for that user. The client can then decide how to handle this - show an error message or not.